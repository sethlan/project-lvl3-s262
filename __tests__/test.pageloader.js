import nock from 'nock';
import pageload from '../src/';

const host = 'http://ru.hexlet.io';
const path = '/';
const answer = { id: 'done' };
nock(host)
  .get(path)
  .reply(200, answer);
test('work for small chunk', () => {
  expect.assertions(1);
  return pageload(host)
    .then(data => expect(data).toEqual(answer));
});
