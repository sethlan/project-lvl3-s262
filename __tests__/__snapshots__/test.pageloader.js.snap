// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test error EISDIR 1`] = `"Error: ENOENT: no such file or directory, open '/folderdnotexist/www-example-com.html'"`;

exports[`test error ENOTFOUND 1`] = `"URL not found, please check address or connection"`;

exports[`test error handle 1`] = `"Error: something awful happened"`;

exports[`test error not correct url 1`] = `"Error: Not correct address"`;

exports[`test error not have args 1`] = `
"Error: Don't have one of arguments:
URL = http://www.example.com Path = "
`;

exports[`test error statuscode 201 1`] = `"Error: statuscode 201"`;
